README.md

```diff
@@ -1,2 +1,22 @@
 # sgbouncewizard
+
 UCI Capstone 2018-19 bounce wizard
+
+## Setup
+
+Before running, make sure you have two environmental files are the root of the project.
+File1: .env
+Contains: API_URL=http://localhost:3000
```

Comment:
Hmm for any other secrety things like API keys/passwords we won't add onto this list of contains:. Maybe we can put a note instead for other secret things please contact the team members for more information.